content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,92 @@
+## Installation
+
+- **Added** a command line [Istio operator](docs/setup/install/operator/) to install and manage the Istio control plane.
+
+## Traffic Management
+
+- **Added** automatic determination of protocol types. This removes the need for you to explicitly name service ports as `http` or `tcp`.
+- **Added** a mode to the Gateway API for mutual TLS operation. The new mode can read certificates via SDS.
+- **Added** Sidecar proxy readiness checks now check Envoy's readiness status.
+- **Fixed** issues present when a service communicates over the network first in permissive mutual TLS mode for protocols like MySQL and MongoDB.
+- **Improved** container ports are no longer required in the pod spec. All ports will be captured by default.
+- **Improved** the `EnvoyFilter` API. You can now add or update all configurations.
+- **Improved** the Redis load balancer to now default to [`MAGLEV`](https://www.envoyproxy.io/docs/envoy/v1.6.0/intro/arch_overview/load_balancing#maglev) when using the Redis proxy.
+- **Improved** load balancing to direct traffic to the [same region and zone](docs/ops/traffic-management/locality-load-balancing/) by default.
+- **Improved** Pilot performance. The reduction in CPU utilization could be up to 10 times depending on deployment.
+- **Improved** the `ServiceEntry` API to allow for the same hostname in different namespaces.
```

Comment:
Link to issue: https://github.com/istio/istio/issues/13008

Reply:
I don't think there is no new doc on how to accomplish it, you just make a ServiceEntry with the same hostname in different namespaces. Before it did not work and now it does.

Reply:
@howardjohn are there docs explaining the prior limitation?  If so, the solution would be to simply remove that limitation from those docs.  If not, i'm not sure adding a link for each bug fix makes sense - atleast for 1.3 given the chaotic nature of our release notes gathering process.

Reply:
There is no doc - we generally capture bugs as github issues :slightly_smiling_face: If we can't link to that then I will resolve this

Reply:
> I don't think there is no new doc on how to accomplish it, you just make a ServiceEntry with the same hostname in different namespaces. Before it did not work and now it does.

Can we get a sum up of the re-solution of the main issue (google points people there)? I read the design doc + all the comments on the issue; yet I am still confused if defining SE with same hostname on different namespaces will cause problems.

The design doc is still a WIP; so is _this_ in 1.3 or not; what exactly was _improved_ here?

Reply:
Not sure which design doc you are refering to, https://docs.google.com/document/d/15-PU9O22Pb0qTzCfwK2hjwNTcb-CqFuw8JnIgwK3EIM/edit#heading=h.7zgnj8bwqfld is approved and implemented (as noted here). You can define a SE with same hostname and different namespace if you have a Sidecar (https://istio.io/docs/reference/config/networking/v1alpha3/sidecar/) selecting only a single one of those for each workload.

Reply:
Wow stellar speed in responding :). Yes, https://docs.google.com/document/d/15-PU9O22Pb0qTzCfwK2hjwNTcb-CqFuw8JnIgwK3EIM is the one I read. I am probably confused about `Status: WIP`.

---

> ...selecting only a single one of those for each workload.

So it won't work _properly_ using the default/stock Sidecar. Please do correct me if I am wrong.

Reply:
The Status field has `approved` bolded

Depends what you mean by properly. You are setting the same thing (hostname) to two different values, so there isn't much that can be done. There are basically two options: merging the two (basically impossible in this case), or picking one of them. We went with picking one, whichever one is the oldest. If you want to select which one you want, use a Sidecar.

Reply:
Oh right; it makes a bit more sense now - it was approved but later on there are no _approvers_ (`Approvers: xxx [], xxx []`) so that's what is confusing.

---

Makes sense. Thank you for clarifying ❤️ .